ext/mysqli/mysqli.stub.php

```diff
@@ -110,111 +110,109 @@ public function connect(
         ?string $database = null,
         ?int $port = null,
         ?string $socket = null
-    ) {}
+    ): mysqli|null|false {}
 
     /**
-     * @return bool
+     * @tentative-return-type
      * @alias mysqli_dump_debug_info
      */
-    public function dump_debug_info() {}
+    public function dump_debug_info(): bool {}
 
     /**
      * @return bool
      * @alias mysqli_debug
      * @no-verify Should really be a static method
      */
-    public function debug(string $options) {}
+    public function debug(string $options) {} // TODO make return type void
 
     /**
-     * @return object|null
+     * @tentative-return-type
      * @alias mysqli_get_charset
      */
-    public function get_charset() {}
+    public function get_charset(): ?object {}
 
     /**
-     * @return string|null
+     * @tentative-return-type
      * @alias mysqli_get_client_info
      * @deprecated 8.1.0
      */
-    public function get_client_info() {}
+    public function get_client_info(): string {}
 
 #if defined(MYSQLI_USE_MYSQLND)
     /**
-     * @return array
+     * @tentative-return-type
      * @alias mysqli_get_connection_stats
      */
-    public function get_connection_stats() {}
+    public function get_connection_stats(): array {}
 #endif
 
     /**
-     * @return string
+     * @tentative-return-type
      * @alias mysqli_get_server_info
      */
-    public function get_server_info() {}
+    public function get_server_info(): string {}
 
     /**
-     * @return mysqli_warning|false
+     * @tentative-return-type
      * @alias mysqli_get_warnings
      */
-    public function get_warnings() {}
+    public function get_warnings(): mysqli_warning|false {}
 
-    /**
-     * @return null|false
-     */
+    /** @return bool|null */
```

Comment:
I used an invalid return type before. I'm not declaring the tentative return, since the new return type still doesn't make much sense.

Reply:
This whole method just doesn't make sense :) It's already on the deprecations list: https://wiki.php.net/rfc/deprecations_php_8_1#mysqliinit